docs/mocksupport.rst

```diff
@@ -878,3 +878,58 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock InMemory CIM Repository`:
+
+Mock in-memory CIM repository
+-----------------------------
+
+pywbem_mock provides a standard interface to a CIM repository that
+stores CIM classes, CIM instances, and CIM qualifier declarations.
+
+This repository implements an repository in memmory that is created and
+destroyed each time the FakedWBEMConnection class is constructed and
+destroyed.
+
+
+.. automodule:: pywbem_mock._inmemoryrepository
+
+.. autoclass:: pywbem_mock.InMemoryRepository
+   :members:
+
+   .. rubric:: Methods
+
+   .. autoautosummary:: pywbem_mock.InMemoryRepository
+      :methods:
+      :nosignatures:
+
+   .. rubric:: Attributes
+
+   .. autoautosummary:: pywbem_mock.InMemoryRepository
+      :attributes:
+
+   .. rubric:: Details
+
+.. _`Mock InMemory Object store`:
+
+Mock InMemory object store
+--------------------------
+
+.. automodule:: pywbem_mock._inmemoryrepository
```

Comment:
Copy&Paste error in "pywbem_mock._inmemoryrepository"

Reply:
Done., I think

Reply:
Sorry for my comment, it seems I did not notice that both classes are in the same module.

What I was stumbling on was that the same module docstring (of `pywbem_mock._inmemoryrepository`)  is used twice.

An additional issue is that some of the intro text is written in this .rst file, and some comes from the module docstring.

Reply:
I suggest to remove this `automodule` directive. It does not provide much value and also it applies to the whole module while this section is scoped to just one class of the module.

Reply:
CLOSED - became moot since the last commit changed the module docstring.